### PR TITLE
lit: add page

### PR DESCRIPTION
**Pull request by P2Tree**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):** LLVM 19

### CODE REVIEW

File: pages/common/lit.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# lit
+
+> LLVM integrated tester for executing LLVM and Clang style test suites, summarizing results.
+> Part of LLVM.
+> More information: <https://www.llvm.org/docs/CommandGuide/lit.html>.
+
+- Run a specified test case:
+
+`lit {{test_file.test}}`
```
Comment (spageektti) [inline pages/common/lit.md:9]:
```suggestion
`lit {{path/to/test_file.test}}`
```

File: pages/common/lit.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+`lit {{path/to/test_suite}}`
+
+- Run all test cases and check the wall time for each cases, then report to summary output:
+
+`lit {{path/to/test_suite}} --time-tests`
+
+- Run individual tests with valgrind (memory check and memory leak test):
+
+`lit {{test_file.test}} --vg --vg-leak --vg-args={{args_to_valgrind}}`
```
Comment (spageektti) [inline pages/common/lit.md:21]:
```suggestion
`lit {{path/to/test_file.test}} --vg --vg-leak --vg-args={{args_to_valgrind}}`
```

**Review by spageektti — COMMENTED** [review]
Hi. Thanks for your contribution!

**Review by gutjuri — APPROVED** [review]
LGTM after the suggestions

**Review by spageektti — APPROVED** [review]

**Pull request merged**